.github/workflows/check_code_format.yml

```diff
@@ -22,8 +22,8 @@ jobs:
 
       - name: Format code
         run: |
-          git clean -f -x -d
-          nim prettyfy
+          git clean -f -x -d -e ".nim_runtime"
+          nim prettyfy --excludePath=".nim_runtime"
```

Comment:
Why not just to run the `setup-nim-action` with [`nim-install-directory`](https://github.com/jiro4989/setup-nim-action/blob/1b7e2433060960eab05e25faa2c9cf1ff69b8cfa/action.yml#L12C3-L12C24) set to some directory _outside_ the workspace?

Reply:
Due to an issue in setup-nim-action apparently: https://github.com/jiro4989/setup-nim-action/issues/499